app/views/travel_requests/_form.html.erb

```diff
@@ -11,6 +11,41 @@
     </div>
   <% end %>
 
+  <div class="field">
+    <%= form.label :creator_id %>
+    <%= form.text_field :creator_id %>
+  </div>
+
+  <div class="field">
+    <%= form.label :start_date %>
+    <%= form.text_field :start_date %>
+  </div>
+
+  <div class="field">
+    <%= form.label :end_date %>
+    <%= form.text_field :end_date %>
+  </div>
+
+  <div class="field">
+    <%= form.label :request_type %>
+    <%= form.text_field :request_type %>
+  </div>
+
+  <div class="field">
+    <%= form.label :purpose %>
+    <%= form.text_field :purpose %>
+  </div>
+
+  <div class="field">
+    <%= form.label :participation %>
+    <%= form.text_field :participation %>
+  </div>
+
+  <div class="field">
+    <%= form.label :travel_category %>
```

Comment:
this will be added during the approval form; not by the initiating requestor. we might want to go ahead and take it off of here.